### DISCUSSION TITLE
missing assembly token

### DISCUSSION THREAD

**Discussion by Damian2591** [iotaledger/firefly#7297] [Help & Support]
since last staking periode started i logged in today and suddenly all my assembly tokens are gone. where are they, how ca i find them?
in my portable wallet in smartphone (android, i know it is not supported anymore) are still all the tokens i get from staking.

**Comment by begonaalvarezd** [comment]
Hi, the node can fail sometimes and take a while to display your rewards, but dont worry your rewards are not gone. If you are using Firefly in multiple devices, in the device where the rewards are not displayed correctly you can run a balance search, please go to settings > advances > balance finder, and run it multiple times, to help Firefly findings addresses where your rewards might be sitting. 
Hope this helps 🙏🏼

**Reply by Damian2591** [reply]
thank you very much, everything is fine again!
